scripts/regen-staticserve-fixture.sh

```
@@ -0,0 +1,98 @@
+# Requirements: npx (offline npm cache of @boundaryml/baml@0.223.0), goimports,
+# gofmt, a Go toolchain, and the cached BAML CFFI (for the CGO adapter regen).
+#
+# Usage (from repo root):
+#   scripts/regen-staticserve-fixture.sh
+set -euo pipefail
+
+BAML_VERSION="0.223.0"
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+cd "$ROOT"
+
+export PATH="$PATH:$(go env GOPATH)/bin"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)